src/main/java/org/opentripplanner/routing/edgetype/TransitBoardAlight.java

```diff
@@ -161,6 +161,12 @@ public State traverse(State s0, long arrivalTimeAtStop) {
             // arrives/departs, so previousStop is direction-dependent.
             s1.setPreviousStop(getStop()); 
             s1.setLastPattern(this.getPattern());
+            int alightTime = options.getAlightTime(this.getPattern().mode);
+            if (alightTime != 0) {
+                s1.incrementTimeInSeconds(alightTime);
+                s1.incrementWeight(alightTime * options.waitReluctance);
+                // TODO: should we have different cost for alighting and boarding?
```

Comment:
Or rather separating waitReluctance and something like boardingAlightingReluctance. The first one is about actually waiting for your transport, while the second is for you getting ready to board or going trough the airport after landing.